README.md

```diff
@@ -1,205 +1,99 @@
-
-#### Homebrew (Mac OS)
-
-```bash
-brew update && brew install meilisearch
-meilisearch
-```
-
-#### Docker
-
-```bash
-docker run -p 7700:7700 -v "$(pwd)/meili_data:/meili_data" getmeili/meilisearch
-```
-
-#### Announcing a cloud-hosted Meilisearch
-
-Join the closed beta by filling out this [form](https://meilisearch.typeform.com/to/VI2cI2rv).
-
-#### Try Meilisearch in our Sandbox
-
-Create a Meilisearch instance in [Meilisearch Sandbox](https://sandbox.meilisearch.com/). This instance is free, and will be active for 48 hours.
-
-#### Run on Digital Ocean
-
-[![DigitalOcean Marketplace](assets/do-btn-blue.svg)](https://marketplace.digitalocean.com/apps/meilisearch?action=deploy&refcode=7c67bd97e101)
+<!-- Go to [this demo]() to try Meilisearch! -->
 
-#### Deploy on Platform.sh
-
-<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/meilisearch/.platform.template.yaml&utm_content=meilisearch&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
-    <img src="https://platform.sh/images/deploy/lg-blue.svg" alt="Deploy on Platform.sh" width="180px" />
-</a>
-
-#### APT (Debian & Ubuntu)
-
-```bash
-echo "deb [trusted=yes] https://apt.fury.io/meilisearch/ /" > /etc/apt/sources.list.d/fury.list
-apt update && apt install meilisearch-http
-meilisearch
-```
-
-#### Download the binary (Linux & Mac OS)
-
-```bash
-curl -L https://install.meilisearch.com | sh
-./meilisearch
-```
-
-#### Compile and run it from sources
-
-If you have the latest stable Rust toolchain installed on your local system, clone the repository and change it to your working directory.
-
-```bash
-git clone https://github.com/meilisearch/meilisearch.git
-cd meilisearch
-cargo run --release
-```
-
-### Create an Index and Upload Some Documents
-
-Let's create an index! If you need a sample dataset, use [this movie database](https://www.notion.so/meilisearch/A-movies-dataset-to-test-Meili-1cbf7c9cfa4247249c40edfa22d7ca87#b5ae399b81834705ba5420ac70358a65). You can also find it in the `datasets/` directory.
-
-```bash
-curl -L https://docs.meilisearch.com/movies.json -o movies.json
-```
-
-Now, you're ready to index some data.
-
-```bash
-curl -i -X POST 'http://127.0.0.1:7700/indexes/movies/documents' \
-  --header 'content-type: application/json' \
-  --data-binary @movies.json
-```
-
-### Search for Documents
+## ✨ Features
 
-#### In command line
+- **Search-as-you-type:** find relevant search results in less than 50 milliseconds
+- **[Typo tolerance](https://docs.meilisearch.com/learn/getting_started/customizing_relevancy.html#typo-tolerance):** get relevant results even when queries contain typos and misspellings
+- **[Filtering and faceted search](https://docs.meilisearch.com/learn/advanced/filtering_and_faceted_search.html#configuring-filters):** enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code
```

Comment:
```suggestion
- **[Filtering and faceted search](https://docs.meilisearch.com/learn/advanced/filtering_and_faceted_search.html):** enhance your user's search experience with custom filters and build a faceted search interface in a few lines of code
```
I think it makes more sense to link to the page about filters and facets instead of linking to the "configuring filters" subsection. It's also more consistent with the rest of the links